prem_utils/default_urls.py

```diff
@@ -0,0 +1,22 @@
+DEFAULT_PREM_URLS = {
```

Comment:
I would move this variable in the `prem.py` file at the class level.

`self.url_mappings = {}`  in the PremConnector constructor.

Reply:
Ahh okay makes sense